### PR TITLE
Debug karma tests

### PR DESCRIPTION
**Pull request by FraukeF**
Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

# CI Check

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (atomicsulfate) [inline .vscode/launch.json:1]: The problem of having this config in github, is that it'll overwrite any custom launch configuration  that you may have locally.  It actually did that silently for me when I pulled this change. Each developer might have their own preferences on how to setup the launch configuration.
We have a section in mapsdk README.md with vscode settings: "Debugging in the browser from Visual Studio Code". Maybe it should better go there?

**Review by atomicsulfate — CHANGES_REQUESTED** [review]

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                "${workspaceFolder}/dist/node_modules/**/*.js"
+            ]
+        },
+        {
+            "type": "chrome",
+            "request": "launch",
+            "name": "Karma Tests",
+            "sourceMaps": true,
+            "webRoot": "${workspaceRoot}/test",
+            "url": "http://localhost:9876/debug.html",
```
Comment (atomicsulfate) [inline .vscode/launch.json:23]: yarn test-debug launches a browser window, and then this launches another one. We could instead attach to the browser already launched by test-debug command.
```suggestion
            "request": "attach",
            "name": "Karma Tests",
            "sourceMaps": true,
            "webRoot": "${workspaceRoot}/test",
            "address": "localhost",
            "port": 9333,
```

File: karma.options.js [hunk 3]
```diff
@@ -41,6 +41,15 @@ const options = function (isCoverage, isMapSdk, prefixDirectory) {
     };
 
     return {
+        browsers: [
+            'ChromeDebug'
+        ],
+        customLaunchers: {
+            ChromeDebug: {
+                base: 'Chrome',
+                flags: ['--no-sandbox', '--remote-debugging-port=9333']
```
Comment (atomicsulfate) [inline karma.options.js:50]:
```suggestion
                flags: ['--no-sandbox', '--remote-debugging-port=9333', 'http://localhost:9876/debug.html']
```

File: .vscode/launch.json (hunk 1 above)
Comment (FraukeF) [reply .vscode/launch.json]: I removed it and added a section to the README.md

File: .vscode/launch.json (hunk 2 above)
Comment (FraukeF) [reply .vscode/launch.json]: done

File: karma.options.js (hunk 3 above)
Comment (FraukeF) [reply karma.options.js]: done

**Review by atomicsulfate — APPROVED** [review]

**Pull request merged**